angular/projects/spark-angular/src/lib/components/sprk-tooltip/sprk-tooltip.component.ts

```diff
@@ -35,13 +41,13 @@ import { uniqueId } from 'lodash';
         <sprk-icon
           [iconType]="iconNameToUse"
           [additionalClasses]="iconAdditionalClasses"
-          aria-hidden="true"
+          ariaHidden="true"
         ></sprk-icon>
       </button>
       <span
         [ngClass]="getTooltipClasses()"
         [attr.id]="id"
-        aria-hidden="true"
+        ariaHidden="true"
```

Comment:
This is a bug since the span is not an angular component and directive. Should revert back to aria-hidden="true"